keras_nlp/models/deberta_v3/deberta_v3_tokenizer.py

```diff
@@ -65,15 +70,29 @@ class DebertaV3Tokenizer(SentencePieceTokenizer):
     ```
     """
 
-    def __init__(self, proto, **kwargs):
+    def __init__(self, proto, mask_token_id=None, **kwargs):
         super().__init__(proto=proto, **kwargs)
 
+        # Maintain a private copy of `mask_token_id` for config purposes.
+        self._mask_token_id = mask_token_id
+
+        # Maintain a private copy of the original vocabulary; the parent class's
+        # `get_vocabulary()` function calls `self.vocabulary_size()`, which
+        # throws up a segmentation fault.
```

Comment:
What is the segmentation fault here? I'm not sure I totally follow. Ideally we don't have to store a copy of the vocabulary. This would be a not-totally-insignificant waste of memory!